### PR TITLE
PlusMinusFinal

### PR DESCRIPTION
**Pull request by leankyr**
Ποιο Σωστό Pull request Αυτό?

### CODE REVIEW

**Review by dimkirt — CHANGES_REQUESTED** [review]
- Don't import unused libraries
- Remove the .txt files, either write tests with a framework like pytest or don't upload any test cases
- Finish the problem first and then we will merge it to master
- Assign a reviewer only if you are done or you need advice, if you need advice change the title to WIP: so that the reviewer knows that this PR is Work In Progress

File: leankyr/Easy_Problems/Plus_Minus/plusMinus.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    with open("testcases/testcase0.txt") as file_obj:
+        data = file_obj.readlines()
+
+    n = int(data[0])
+    print (n)
+    arr = list(map(int, data[1].rstrip().split()))
+
+
+    # array is ready to be used as a list
+    #plusMinus(arr)
```
Comment (dimkirt) [inline leankyr/Easy_Problems/Plus_Minus/plusMinus.py]: so you didn't finish the problem yet right?

**Comment by leankyr** [comment]
this is v 1.0 of PlusMinus implementation

**Comment by leankyr** [comment]
This time I used list comprehensions

**Review by dimkirt — APPROVED** [review]

**Pull request merged**